server/lib/genome/importers/api_importers/civic/importer.rb

```diff
@@ -9,14 +9,87 @@ class Importer < Genome::Importers::Base
 
           def initialize
             @source_db_name = 'CIViC'
+            @api_client = ApiClient.new
           end
 
           def create_claims
+            create_drug_claims
+            create_gene_claims
             create_interaction_claims
           end
 
           private
 
+          # this should be rare -- maybe impossible under the latest schema changes?
+          def importable_drug?(drug)
+            drug.name.upcase != 'N/A' && !drug.name.include?(';')
+          end
+
+          def importable_evidence_item?(ei)
```

Comment:
I think maybe the logic here got swapped around but the name didn't. For instance, wouldn't we want to import evidence with a rating `>= 2`, not `<=`? 

Unless I'm reading it wrong, the [old method](https://github.com/dgidb/dgidb-v5/pull/493/files#diff-7f1159fd04b53c023a0e172db1c5628bdfe738522e5b246e74a831c103aea9a5L41) maybe looks closer to correct?

This look more like its checking if we should skip the evidence, rather than import it.

Reply:
Yup, I flipped around the logic but got tripped up. I think it's simpler the old way.